context.go

```diff
@@ -38,7 +39,8 @@ func (c *Context) Bind(i interface{}) bool {
 		dec := json.NewDecoder(c.Request.Body)
 		err = dec.Decode(i)
 	} else {
-		// TODO:
+		// TODO: add more payload types
+		err = errors.New("Unsupported payload type")
```

Comment:
Would be even better to have error similar to net/http - https://golang.org/pkg/net/http/#pkg-variables. `ErrUnsupportedPayload` so a comparison can be done by the caller.